css/index.css

```diff
@@ -112,6 +135,46 @@ p {
     justify-content: center;
 }
 
+.middle-content .boxes .box:nth-child(1){
```

Comment:
For the selector you could have done `box:nth-child(1)` without the other classes since you're not using the box class outside of those classes anywhere